VACMS-10845 Facilities runbook updates

jilladams opened the pull request:
## Description

Relates to https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10845
Relates to https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13195

Today I updated 9 existing facility runbook tickets to use information from the latest runbooks, as a learning exercise for myself. In the process, I found a few things happening that I opted to standardize here. Feel free to modify this branch directly or propose changes: 
1. **Redirect deploy notes** were vague. More specific data is useful for building knowledge, so I updated the "Redirects are usually released" language to be specific re: timing / process.
   * Where the templates include a "execute on the redirect" step, I have moved the deploy related notes to follow that step, in almost all cases. Exceptions are noted in PR comments. 
3. **Facility API ID**: Old templates asked for the Facility Locator URL for changes to existing facilities. New ones don't. I added it back in as `Facility API ID` since team identified that the API ID is the source of truth for the entity in VAST.
4. **Labels:** Added team, product, and discipline labels where they were missing.
5. **Team checkboxes:** removed
6. **Drupal admin steps**: For runbooks where the final batch of "CMS Engineer" steps are not in code but in Drupal, updated ownership to say "Drupal Admins (CMS Engineers or Help desk)" for Drupal steps that _follow_ redirect implementation. This doesn't fully satisfy the requirements for #13195 but is a step toward satisfying that.

### Definition of Done

- [x] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [x] `Facilities`
- [ ] `User support`


### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

swirtSJW commented on .github/ISSUE_TEMPLATE/runbook-vamc-facility-closed.md:
```diff
@@ -60,17 +61,19 @@ If this facility has been removed from VAST in error, please notify our Support
-- [ ] 5. Create a [redirect request](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/new?assignees=&labels=Redirect+request&template=redirect-request-facility-url.md&title=Redirect+Request+for%3A+%3Cinsert+facility+name%3E) from the URL of the facility to be closed to the URL of its parent system.
+- [ ] 5. Create a redirect request, from the URL of the facility to be closed to the URL of its parent system.
+
+https://github.com/department-of-veterans-affairs/va.gov-cms/issues/new?assignees=&labels=Redirect+request&template=redirect-request-facility-url.md&title=Redirect+Request+for%3A+%3Cinsert+facility+name%3E
```
Was removing this from a shorter link tag intentional?

jilladams replied:
Yep. When input that way, in GH front-end, these were rendering as markdown, rather than rendering as a link. After playing with it, it only renders this link as a link if it's only its own line with an empty line above it. (I think it may be due to query parameters.)

ryguyk replied:
This seems to work, at least in this comment.
- [ ] 5. Create a [redirect request](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/new?assignees=&labels=Redirect+request&template=redirect-request-facility-url.md&title=Redirect+Request+for%3A+%3Cinsert+facility+name%3E) from the URL of the facility to be closed to the URL of its parent system.

swirtSJW commented on .github/ISSUE_TEMPLATE/runbook-vamc-facility-new.md:
```diff
@@ -49,11 +50,14 @@ Please let us know when your draft content is complete, so that we can wrap up t
 
 </details>
-- [ ] 5. Create a [URL change request](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/new?assignees=&template=runbook-facility-url-change.md&title=URL+Change+for%3A+%3Cinsert+facility+name%3E), changing the entry from the old facility URL to the new facility URL. (**Note: The URL change request ticket blocks the completion of this ticket.**)
+- [ ] 5. Create a [URL change request, changing the entry from the old facility URL to the new facility URL.
```
```suggestion
- [ ] 5. Create a URL change request, changing the entry from the old facility URL to the new facility URL.
```

omahane reviewed (approved):
I think these are a great improvement. Thanks for looking at this. I had one minor edit, which I must have overlooked when I worked on these last time.

jilladams commented:
Reverted the commit for markdown changes to the redirect ticket link based on convo w Ryan here, and added a one-line change to fix the new VAMC template where the issue did repro / is fixed by a blank line after `</details>`: https://dsva.slack.com/archives/C0FQSS30V/p1681771043842529?thread_ts=1681502414.393989&cid=C0FQSS30V.

ryguyk commented on .github/ISSUE_TEMPLATE/runbook-vamc-facility-name-change.md:
```diff
@@ -20,8 +20,9 @@ Submitter: <insert_name>
 KB articles: <insert_kb_article_links>
 
-- [ ] Link to facility in production:
+- [ ] Link to facility in production, including facility locator link:
 Facility link: <insert_facility_link>
+Facility locator link: <insert_facility_locator_link>
```
I see this change in a number of places. I'm not in love with it, personally. Here's why:
1. The wording makes it sound like the facility-locator link is somehow a component of the other link ("including").
2. I'm actually not sure what I'm supposed to put here.  Is the "Link to facility in production" a va.gov link or a CMS link?
3. It's not immediately clear what "Facility locator link" means. There's a placeholder page that is created for facilities that are in VAST and/or Lighthouse but not yet published in Drupal, and that page is the one linked by the facility locator _initially_. But after the CMS page is published, a URL Change Request would have changed this so that the facility locator now links to the CMS-generated page. So, in many cases (including, specifically, this case, a facility name change) the page linked by the facility locator _is_ the va.gov production link. Technically, the original placeholder page still exists, but a redirect would prevent anybody from actually getting there. So I doubt that URL would be useful in this context. So, what exactly are we asking for here and why?

jilladams replied:
Fair. I did that from a place of ignorance mostly because it used to exist in some templates, and get populated. I, again, am ignorant. Happy to remove it again if it doesn't actually make sense.

jilladams replied:
@ryguyk is this a blocker for approval from your perspective?

jilladams replied:
I talked to Christian and understand better now that this is really standing in for Facility API ID. I'll either remove this or change it to be explicit on that.

jilladams commented on .ddev/mutagen/mutagen.yml:
```diff
@@ -2,9 +2,8 @@
-# Please do `ddev mutagen reset` after changing the file.
 # See ddev mutagen docs at
-# https://ddev.readthedocs.io/en/latest/users/install/performance/
+# https://ddev.readthedocs.io/en/latest/users/performance/#using-mutagen
```
not sure if I should manually edit this value back to what it was.

omahane replied:
Well this link returns a 404, so probably not a good change.
```suggestion
# https://ddev.readthedocs.io/en/latest/users/install/performance/
```

jilladams commented on .github/ISSUE_TEMPLATE/runbook-vamc-facility-closed.md:
```diff
@@ -70,20 +70,17 @@ If this facility has been removed from VAST in error, please notify our Support
-Please check the team(s) that will do this work.
+#### Drupal Admin steps (CMS Engineer or Helpdesk)
+_Help desk will complete these steps or escalate to request help from CMS engineering._
+- [ ] 7. When redirect has been deployed, make these changes. Practice first on staging or a demo environment.
```
Because we added step 6 to execute the redirect steps, the old 6, 6a, 6b etc became 7, 7a, 7b

jilladams commented on .github/ISSUE_TEMPLATE/runbook-vamc-facility-closed.md:
```diff
@@ -70,20 +70,17 @@ If this facility has been removed from VAST in error, please notify our Support
-  - [ ] 6b. CMS engineer edits the facility node, removes flag `Removed from source`, add a revision log that explains the change, with a link to github issue, and change moderation state to archive. (Note: any remaining health services, non-clinical services and events for the given facility will be archived automatically when these changes are saved.)
-  - [ ] 6c. CMS engineer finds the menu for the system https://prod.cms.va.gov/admin/structure/menu and deletes the menu item for the facility being closed.
+#### CMS Engineer steps
+- [ ] 6. Execute the steps of the URL change request ticket from step 5 above.
```
New addition to this runbook. Previously we went straight from "request redirect" to "when redirect has been deployed" without a step to implement

jilladams commented on .github/ISSUE_TEMPLATE/runbook-vamc-facility-new.md:
```diff
@@ -49,28 +50,24 @@ Please let us know when your draft content is complete, so that we can wrap up t
-- [ ] 10. Let help desk know this has been done.
+- [ ] 7. CMS engineer executes the steps of the URL change request ticket from step 5 above.
 
+(Redirects deploy weekly on Wed. at 10am ET, or by requesting OOB deploy (of the revproxy job to prod) in #vfs-platform-support. Coordinate the items below and canonical URL change after URL change ticket is merged, deployed, and verified in prod.)
```
Relocated deploy info below the task which will cause deploy.

jilladams commented on .github/ISSUE_TEMPLATE/runbook-vamc-facility-duplicate-or-section-change.md:
```diff
@@ -39,10 +40,12 @@ If there has been a section change, which results in a change to the facility UR
 
-(Redirects are released Wednesday afternoons, so coordinate the following items below and canonical URL change around that timeframe.)
+
+(Redirects deploy weekly on Wed. at 10am ET, or by requesting OOB deploy (of the revproxy job to prod) in #vfs-platform-support. Coordinate the items below and canonical URL change after URL change ticket is merged, deployed, and verified in prod.)
```
deliberately did _not_ relocate these notes here below step 2 where change request gets implemented, because this runbook is more complex with more Engineer steps following.

jilladams commented on .github/ISSUE_TEMPLATE/runbook-vet-center-closed.md:
```diff
@@ -21,32 +21,29 @@ Submitter: <insert_name>
 
-#### CMS Help desk (wrap up)
-- [ ] 7. Help desk notifies editor and any other stakeholders.
+(Redirects deploy weekly on Wed. at 10am ET, or by requesting OOB deploy (of the revproxy job to prod) in #vfs-platform-support. Coordinate the items below and canonical URL change after URL change ticket is merged, deployed, and verified in prod.)
```
Added this and step 3 to align with how we're doing this on other runbooks.

jilladams commented:
@omahane @swirtSJW @ryguyk : PR updated with notes from today in scrum / refinement, and updated PR description with what's done here in the end post-churn.

omahane reviewed (changes requested):
After that mutagen change is made, I will approve.

jilladams commented:
@omahane done!

omahane reviewed (approved):
These are great changes and makes the various processes much clearer.
